packages/embed/src/types.ts

```diff
@@ -181,4 +181,19 @@ export type Message = { channel: string; data?: unknown } & (
   | {
       type: 'submitForm'
     }
+  | {
+      type: 'appleAbortSession'
+    }
+  | {
+      type: 'appleCompletePayment'
+      data: boolean
+    }
+  | {
+      type: 'completeMerchantValidation'
```

Comment:
Should this be name scoped to apple like the others? i.e. `appleCompleteMerchantValidation`

Reply:
Yep - Ill make the udpate